model/src/main/java/com/epam/esm/entities/GiftCertificate.java

```diff
@@ -0,0 +1,114 @@
+package com.epam.esm.entities;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class GiftCertificate extends Entity {
+
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private int duration;
+    private LocalDateTime createDate;
+    private LocalDateTime lastUpdateDate;
+
+    public GiftCertificate() {
+    }
+
+    public GiftCertificate(String name, String description, BigDecimal price, int duration) {
```

Comment:
What if create a builder instead of creating 4 types of constructor?
[optional] also you can use `lombok` to avoid creation of millions of getters/setters.